pages/projects/index.tsx

```diff
@@ -28,7 +28,7 @@ const Projects = () => {
     <SharedLayout title="Projects">
       <div className="flex flex-col w-full">
         <AuthModal title={'Continue with your social accounts'} />
-        <div className="sticky top-0 z-10 backdrop-blur-3xl">
+        <div className="sticky top-0 z-10 backdrop-blur-custom">
```

Comment:
```suggestion
        <div className="sticky top-0 z-10 backdrop-blur-[2px]">
```
You can try like this. In my opinion, you don't need to add in `tailwind config file`.